public/components/content-list-item/templates/needsPictureDesk.html

```diff
@@ -0,0 +1,6 @@
+<td class="content-list-item__field--needs-picture-desk" title="Needs Picture Desk: {{ contentItem.needsPictureDesk.toLowerCase() }}">
+
+    <i ng-if="contentItem.needsPictureDesk == 'REQUIRED' || contentItem.needsPictureDesk == 'COMPLETE'" class="content-list-item__icon--incopy content-list-item__icon--inline" wf-icon="needs-picture-desk"></i>
```

Comment:
Not strictly related to this PR, but is it worth switching out `content-list-item__icon--incopy` for something like `content-list-item__icon--needs...`? At the minute, both this and `needsLegal` are using styling that seems to be tied to another icon.

Reply:
Done